src/pybind/mgr/devicehealth/module.py

```diff
@@ -220,6 +220,11 @@ def config_notify(self):
                     self.get_module_option(opt['name']))
             self.log.debug(' %s = %s', opt['name'], getattr(self, opt['name']))
 
+    def notify(self, notify_type, notify_id):
```

Comment:
osdmap updates can be quite frequent - if this is just so the pool gets created, it'd be more efficient to have an instance variable tracking whether the pool still needs creation, so we can avoid calling check_health() for every new map